### PR TITLE
Move SwaggerJsonConverter out of project

### PR DESCRIPTION
**Pull request by harageth**
# Resolves

    JIRA-330

# What

    Migrates the swaggerJsonConverter out of this project and changes the pom to be able to reference the new location

## How to test

    You should just need to Compile this with the salus-tools repo checked out after it is compiled.

# Why

    The tool needed to be used from multiple locations so it needed to be moved to a different repo

# TODO

    NA

### CODE REVIEW

File: templates/markdown.hbs [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+
+## {{join schemes " | "}}://{{host}}{{basePath}}
+
+
+{{description}}
+
+{{#contact}}
+[**Contact the developer**](mailto:{{email}})
+{{/contact}}
```
Comment (jjbuchan) [inline templates/markdown.hbs]: The contact details should probably be to contact RS support?

File: pom.xml [hunk 2]
```diff
@@ -130,6 +146,72 @@
...
+            <apiSource>
+              <springmvc>true</springmvc>
+              <locations>
+                <location>com.rackspace.salus.resource_management.web.model</location>
+                <location>com.rackspace.salus.resource_management.web.controller.ResourceApi</location>
+              </locations>
+              <templatePath>${basedir}/templates/strapdown.html.hbs</templatePath>
+              <schemes>
+                <scheme> https </scheme>
+                <scheme> http </scheme>
```
Comment (jjbuchan) [inline pom.xml]: Still curious whether this should have spaces?

Comment (harageth) [reply pom.xml]: It was a copy paste. I will fix it on both of them.

File: templates/markdown.hbs [hunk 3]
```diff
@@ -7,7 +7,7 @@
 {{description}}
 
 {{#contact}}
-[**Contact the developer**](mailto:{{email}})
+[**Contact the developer**](mailto:contact your support team)
```
Comment (itzg) [inline templates/markdown.hbs]:
```suggestion
[**Contact your support team**](mailto:{{email}})
```

**Pull request merged**